### PR TITLE
Ankit | Test | Test - VAT liabilities and VAT payment option is not coming in cmd+k and all items page

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cweaquz
https://app.clickup.com/t/86cweb63w
https://app.clickup.com/t/86cwebf1e

### CODE REVIEW

File: apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts [hunk 1]
```diff
@@ -153,7 +150,41 @@ export class VatLiabilitiesPayments implements OnInit, OnDestroy {
...
+                }
+                if (this.isCompanyMode) {
+                    this.hasTaxNumber = true;
+                }
+                this.getLiabilitiesPayment();
+            }
+        });
+    }
+        /**
+     * Get Current company branches information
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts:168]: format code

File: apps/web-giddh/src/assets/locale/all-items/en.json [hunk 2]
```diff
@@ -552,6 +552,34 @@
...
                     }
+                },
+                {
+                    "label": "VAT Liabilities",
+                    "link": "/pages/vat-report/liabilities",
+                    "icon": "icon-vat1",
+                    "description": "VAT Liabilities",
+                    "hide": "sidebar",
+                    "additional": {
+                        "queryParams": {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/assets/locale/all-items/en.json:563]: As confirmed with kriti please do for voucher version 2 only

File: apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html [hunk 3]
```diff
@@ -87,15 +87,19 @@
                                 {{ commonLocaleData?.app_from }}
                             </th>
                             <td mat-cell *matCellDef="let element">
-                                {{ element?.taxPeriod?.from ?? "-" }}
+                                {{
+                                    element?.taxPeriod?.from
+                                        ? dayjs(element.taxPeriod.from).format(giddhDateFormat)
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html]: revert code for date format in overall file and assign to api developer as confirmed with kriti

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts:168]: Done

File: apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.html]: Done

File: apps/web-giddh/src/assets/locale/all-items/en.json (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/assets/locale/all-items/en.json:563]: only for 2

**Review by dvCodeWorld — APPROVED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**